src/app/econ/supply-demand/lesson/SupplyDemandLesson.tsx

```diff
@@ -0,0 +1,748 @@
+          the price of the good.
+          <br />
+          <br />
+          The <Emphasize>Demand curve</Emphasize> is the line where it is
+          sloping downwards and the <Emphasize>Supply curve</Emphasize> is the
+          line where it is sloping upwards.
+          <br />
+          <br />
+          Now let's dig deeper to see why this is the case:
+        </p>
+      </ColorBox>
+    </Block>,
+
+    <Block color="blue" title="Demand">
+      <p>
+        Demand means how much people want to buy something.
+        <br />
+        <Emphasize>👉 Example: </Emphasize> If ice cream is cheap on a hot day,
+        lots of people want it!{" "}
+      </p>
+      <ColorBox color="blue">
+        <p>
+          The law of demand states that{" "}
+          <Emphasize>
+            as price goes up, the amount that people want to buy
+          </Emphasize>{" "}
+          (Quantity Demanded) <Emphasize> goes down</Emphasize>, and vice versa.
+        </p>
+        <div>
+          <Emphasize>
+            <u>Reasons: </u>
+          </Emphasize>
+          <ul>
+            <li>
+              <Collapsible
+                ExampleContent={
+                  <>
+                    <Emphasize>👉 Example: </Emphasize> If ice cream is too
+                    expensive, they will buy a candy bar instead
+                  </>
+                }
+              >
+                <Emphasize>- Substitution effect: </Emphasize> Consumers switch
+                to alternatives.
+              </Collapsible>
+            </li>
+            <li>
+              <Collapsible
+                ExampleContent={
+                  <>
+                    <Emphasize>👉 Example: </Emphasize> If they have more money
+                    to spend, they will prefer to get something more expensive
+                    like steak instead of having something that is cheap and
+                    processed like ramen noodles.
+                  </>
+                }
+              >
+                <Emphasize>- Income effect: </Emphasize> Budget constraints
+                change purchasing power.
+              </Collapsible>
+            </li>
+            <li>
+              <Collapsible
+                ExampleContent={
+                  <>
+                    <Emphasize>👉 Example: </Emphasize> If they have more and
+                    more ice cream, they will be less and less satisfied with
+                    each additional ice cream.
+                  </>
+                }
+              >
+                <Emphasize>- Diminishing </Emphasize>
+                <a
+                  href="https://pincs.stanford.edu/pickcode-dev/econ/marginal-utility/?slide=1"
+                  style={{ color: "blue", textDecoration: "underline" }}
+                >
+                  marginal utility:
+                </a>{" "}
+                The satisfaction from buying more of the same thing decreases.{" "}
```

Comment:
I think this last point (**Diminishing**) is still in the collabsible wake of the other component or something...

<img width="832" height="363" alt="Screenshot 2025-07-16 at 8 26 35 PM" src="https://github.com/user-attachments/assets/33b4ac71-1432-405e-aa4f-004c83e31029" />